From a6cf671bccac61510d68ecaca05f935b1994b1ab Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Thu, 6 Jan 2022 23:57:52 +0800
Subject: [PATCH 2/5] Revert "drm/amdgpu: When the VCN(1.0) block is suspended,
 powergating is explicitly enabled"

This reverts commit 28863ffe21ff711d5109e3c208676258bdec3a1f.
---
 drivers/gpu/drm/amd/amdgpu/vcn_v1_0.c | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/vcn_v1_0.c b/drivers/gpu/drm/amd/amdgpu/vcn_v1_0.c
index 7115f6dbb137..e8737fa438f0 100644
--- a/drivers/gpu/drm/amd/amdgpu/vcn_v1_0.c
+++ b/drivers/gpu/drm/amd/amdgpu/vcn_v1_0.c
@@ -254,13 +254,6 @@ static int vcn_v1_0_suspend(void *handle)
 {
 	int r;
 	struct amdgpu_device *adev = (struct amdgpu_device *)handle;
-	bool idle_work_unexecuted;
-
-	idle_work_unexecuted = cancel_delayed_work_sync(&adev->vcn.idle_work);
-	if (idle_work_unexecuted) {
-		if (adev->pm.dpm_enabled)
-			amdgpu_dpm_enable_uvd(adev, false);
-	}
 
 	r = vcn_v1_0_hw_fini(adev);
 	if (r)
-- 
2.27.0

